packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/newLineNameMorphFor..st

```diff
@@ -0,0 +1,10 @@
+private
+newLineNameMorphFor: aTraceValue
+	
+	| name |
+	name := aTraceValue belongingProbeId asString asText.
+	name addAttribute: (BPPluggableTextAttribute new evalBlock: [
```

Comment:
I'm asking myself here, why does the text get the action block and not the morph? with `actionSelector:` `arguments: ` and `target` on a `Morph`, like don in `newHaltButtonFor:`;
````
exploreButton 
		actionSelector: #haltButtonClickedFor:and:;
		arguments: {self id . aTrace example};
		target: self;
		balloonText: 'click to open a debugger at the first invocation of this expression'.
```

Reply:
because the haltButton is a button, while the lineMorph and TitleMorph are currently not.

Reply:
New Solution is now using `Morph on:send:to:` for the on-click action